tools/isobuild/compiler-plugin.js

```diff
@@ -175,6 +177,8 @@ export class CompilerPluginProcessor {
       var sourceProcessor = data.sourceProcessor;
       var resourceSlots = data.resourceSlots;
 
+      self.cancelToken.throwIfRequested();
```

Comment:
The interruptibility of the build process depends on how often `cancelToken.throwIfRequested` gets called. Calling it too often isn't really a problem, but calling it during code that should not be interrupted can be a serious problem.